test/fail/ok/object-ext-var-base.tc.ok

```diff
@@ -0,0 +1,6 @@
+object-ext-var-base.mo:9.11-9.18: type error [M0179], base has var field 
+  v
+object-ext-var-base.mo:9.11-9.18: info, overwrite field to resolve error
+object-ext-var-base.mo:9.11-9.18: type error [M0179], base has var field 
```

Comment:
Buuuh, why do we get 2 error messages?

Reply:
probably because of synthesis context, with
``` diff
index 9a14cd5c8..ccf97b925 100644
--- a/test/fail/object-ext-var-base.mo
+++ b/test/fail/object-ext-var-base.mo
@@ -6,4 +6,4 @@ object no_var {
     public let w = 25
 };
 
-let _ = { has_var and no_var }
+let _ = { has_var and no_var } : {var v : Nat; w : Nat }
```
I only get a single error.

Reply:
resolved by 1a688a28369d809d8ca7af32254b117c5791d41f, thanks @crusso!